src/main/java/org/jabref/gui/Base.css

```diff
@@ -387,6 +387,21 @@
     -fx-padding: -0.1em 0.5em 0.5em 0.5em;
 }
 
+.progress-indicator {
```

Comment:
The progress indicator is also used at other places. Since this definition here is global, it changes also the style of these other indicators, is this desired? I would guess some of the changes are ok globally (e.g the color) but others like the padding or no percentage are probably specific to the toolbar, right?

Reply:
Make sense, I didn't see it anywhere else. I'll see to it.

Reply:
Sounded easy, somehow doesn't work. I am not familiar with javafx, can you spot any mistakes in https://github.com/JabRef/jabref/pull/6443/commits/fcb1d0cc0ec3af744485b4d9321e3a3215c532f1? I tried to copy the style of icon-button which also has multiple styles.
The indicator is invisible now and can only be seen by the changing background color when hovering over it. The area that is then seen is very tiny.

Reply:
I knew you would spot that mistake immediately, thanks! Works like a charm.
https://github.com/JabRef/jabref/pull/6443/commits/e24c1418564a9304b9c5172573e9edc9d1036a54